src/components/dashboard/main-content/DashboardMainContent.jsx

```diff
@@ -46,6 +47,9 @@ const DashboardMainContent = () => {
             >
               Find a course
             </Button>
+
+            <br />
+            <CourseRecommendations />
```

Comment:
We only want to show this if learner has no enrollments.

Reply:
ah...just found that this is inside `<CourseEnrollments>` and it will automatically take care of the use case I mentioned above.